.github/workflows/pytest.yml

```diff
@@ -231,10 +231,10 @@ jobs:
     timeout-minutes: 90
     steps:
       - uses: actions/checkout@v2
-      - name: Set up Python 3.9
+      - name: Set up Python 3.10
         uses: actions/setup-python@v2
         with:
-          python-version: 3.9
+          python-version: "3.10"
```

Comment:
Is the python version bump strictly necessary?

Reply:
@arnavgarg1 It is not; I just felt that testing on the latest -- if all tests pass -- would be better.  What do you think?

Reply:
Fair enough!